components/display/ili9xxx.rst

```diff
@@ -105,14 +103,19 @@ Configuration variables:
    - **mirror_x** (*Optional*, boolean): If true, mirror the x axis.
    - **mirror_y** (*Optional*, boolean): If true, mirror the y axis.
 
-
-**Note:** To rotate the display in hardware use one of the following combinations - with 90 and 270 rotations you
-will also need to use `dimensions:` to swap the height and width (see example below.)
-
+**Note:** The ``rotation`` variable will do a software based rotation. It is better to use the **transform** to rotate the display in hardware. Use one of the following combinations:
     - 90 degrees - use ``swap_xy`` with ``mirror_x``
     - 180 degrees - use ``mirror_x`` with ``mirror_y``
     - 270 degrees - use ``swap_xy`` with ``mirror_y``
 
+With 90 and 270 rotations you will also need to swap the **dimensions** ''height'' and ''width'' (see example below.
+
+
+For modify the SPI setting see :ref:`SPI bus <spi>` . The default **data_rate** is set to ``40MHz`` and the **spi_mode** mode is ``MODE0`` but some displays require ``MODE3`` (*).
+
+**Note:** the default **data_rate** is set to an optimal speed using half-duplex mode (= ''MISO'' not defined) or  
+using the IOMUX pins.
```

Comment:
```suggestion
**Note:** The maximum achievable data rate will depend on the chip type (e.g. ESP32 vs ESP32-S3) the pins used (on ESP32 using the default SPI pins allows higher rates) and the connection type (on-board connections will support higher rates than long cables or DuPont wires.) If in doubt, start with a low speed and test higher rates to find what works. A MISO pin should preferably not be specified, as this will limit the maximum rate in some circumstances, and is not required if the SPI bus is used only for the display.
```